packages/trader/src/Modules/Trading/Containers/trade-chart.tsx

```diff
@@ -115,11 +116,11 @@ const TradeChart = observer((props: TTradeChartProps) => {
             feedCall={{
                 activeSymbols: false,
             }}
-            enabledNavigationWidget={isDesktop()}
+            enabledNavigationWidget={!isMobile}
             enabledChartFooter={false}
             id='trade'
-            isMobile={is_mobile}
-            maxTick={is_mobile ? max_ticks : undefined}
+            isMobile={isMobileShared()}
+            maxTick={isMobile ? max_ticks : undefined}
```

Comment:
Why two different isMobile function?

Reply:
Hi @hasan-deriv, the ismobile from shared has different breakpoint. I need it from screen 900px below to make dtrader asset dropdown not overlapping. Will check if i still need it or not